db/migrate_cards/20151123174009_fix_bad_website_card.rb

```diff
@@ -0,0 +1,19 @@
+# -*- encoding : utf-8 -*-
+
+class FixBadWebsiteCard < Card::Migration
+  def up
+    bad_website_cards =
+      Card.where("type_id = #{Card::PointerID} and name like '%+website'")
+      .joins('LEFT JOIN card_actions on cards.id = card_actions.card_id')
```

Comment:
Place the . on the previous line, together with the method call receiver.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.